answer/positiveAnswerGeneral.json

```diff
@@ -0,0 +1,10 @@
+{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id": "http://pop2020.dedis.ch/answer/positiveAnswerGeneral.json",
+	"title": "All requests (except catchup (list of message) and propagate message (nothing)) expect an ACK (0) as answer",
+	"description": "[Int] success result message",
+
+	"type": "integer",
+	"minimum": 0, "maximum": 0, 
```

Comment:
Using constant value?
https://json-schema.org/understanding-json-schema/reference/generic.html#constant-values

Reply:
Well spotted! I thought `const` only applied to strings!